app/presenters/api/applications_new_presenter.rb

```diff
@@ -0,0 +1,30 @@
+# frozen_string_literal: true
+
+class Api::ApplicationsNewPresenter
+  include Applications
+  include PlansHelper
+  include System::UrlHelpers.system_url_helpers
```

Comment:
I would prefer doing https://github.com/3scale/porta/pull/2522/files#r673858878 rather than repeating everytime when you include Applications